src/modules/Channel/components/MessageInputWrapper/index.tsx

```diff
@@ -37,22 +36,27 @@ export const MessageInputWrapper = (props: MessageInputWrapperProps) => {
   return (
     <MessageInputWrapperView
       {...props}
-      disabled={disableMessageInput}
       {...context}
+      disabled={disableMessageInput}
       currentChannel={currentGroupChannel}
       quoteMessage={context.quoteMessage}
-      sendUserMessage={(params) => (
-        sendMessage({ ...params, quoteMessage: { messageId: params.parentMessageId } as SendableMessageType })
-      )}
-      sendFileMessage={(params: FileMessageCreateParams) => (
-        sendFileMessage(params.file as File, { messageId: params.parentMessageId } as SendableMessageType)
-      )}
-      sendVoiceMessage={({ file, parentMessageId }: FileMessageCreateParams, duration: number) => (
-        sendVoiceMessage(file as File, duration, { parentMessageId } as SendableMessageType)
-      )}
-      sendMultipleFilesMessage={({ fileInfoList, parentMessageId }) => (
-        sendMultipleFilesMessage(fileInfoList.map((fileInfo) => fileInfo.file) as File[], { parentMessageId } as SendableMessageType)
-      )}
+      sendUserMessage={(params) => {
+        return sendMessage({
+          message: params.message,
+          mentionTemplate: params.mentionedMessageTemplate,
+          mentionedUsers: params.mentionedUsers,
+          quoteMessage: context.quoteMessage,
```

Comment:
Why don't we get `quoteMessage` from context using an object destructuring assignment?

```tsx
const {
  ...,
  quoteMessage,
} = context;
```

Reply:
There's no particular reason! 😄

Reply:
addressed in https://github.com/sendbird/sendbird-uikit-react/pull/971/commits/da2a4d3caca8a42c3b4a72e58f35d7dbd061c697